### ISSUE TITLE
Unable to resolve interface type 'Android.Gms.Common.Apis.GoogleApiClient/IConnectionCallbacks

### ISSUE THREAD

**Issue by manumohant** [domaven/xamarin-plugins#33]
Hi I'm facing an issue

> Severity	Code	Description	Project	File	Line	Suppression State
Error		Unable to resolve interface type 'Android.Gms.Common.Apis.GoogleApiClient/IConnectionCallbacks'. Are you missing an assembly reference?	ProjectName.Android	C:\Users\rendy\Maven\xamarin-plugins\Geofence\Geofence\Geofence.Plugin.Android\GeofenceImplementation.cs

**Comment by jochembroekhoff** [comment]
Yes, same issue here. I solved it by changing the target to Android 8.0 (Oreo), updating all the nuget packages and then recompiling the project.
This works, but it might break a lot of projects that are dependent on lower Google Play Services versions.

**Comment by fontalland** [comment]
Same issue here. I tried what the @jochembroekhoff told, but here, even creating a new project, installing the geofence package and updating all the packages with the target to Android 8, the problem persist.
Does anyone know how to solve it?

**Comment by io-engineers** [comment]
Same issue here. When will this be solved? Some packages require a Google Playservices > 42.xxx version, so I cannot stay at version < 27.xxx any longer. Please find solution! Thanks.

**Comment by marcelo-g-simas** [comment]
We submitted a couple pull requests that address this and the authors of this package released a new plugin that incorporate them.  Try https://www.nuget.org/packages/Plugin.Geofence/, it is almost a complete drop in replacement with the exception of a namespace change.
